### PR TITLE
samples: sensor: doc: fix a code block formatting issue

### PR DESCRIPTION
**Pull request by mshinjo**
This commit fixes a code block formatting issue.

Signed-off-by: Mirai Shinjo <miraishinjo@icloud.com>

### CODE REVIEW

File: samples/sensor/stm32_vbat_sensor/README.rst [hunk 1]
```diff
@@ -47,4 +47,4 @@ Sample Output
 
 .. code-block:: console
 
-Current Vbat voltage: 3.04 V
+    Current Vbat voltage: 3.04 V
```
Comment (MaureenHelm) [inline samples/sensor/stm32_vbat_sensor/README.rst:50]: This fix isn't quite right. You've added one space too many

**Review by MaureenHelm — CHANGES_REQUESTED** [review]

File: samples/sensor/stm32_vbat_sensor/README.rst (hunk 1 above)
Comment (mshinjo) [reply samples/sensor/stm32_vbat_sensor/README.rst]: Thanks for the correction. I have force pushed the new change.

**Review by MaureenHelm — CHANGES_REQUESTED** [review]
Will you please update the commit message to mention the subsystem? Something like this:
```
samples: sensor: doc: fix a code block formatting issue

This commit fixes a code block formatting issue in the stem32_vbat_sensor
sample documentation.

Signed-off-by: Mirai Shinjo <miraishinjo@icloud.com>
```

**Comment by mshinjo** [comment]
I have updated the commit message.

**Review by MaureenHelm — APPROVED** [review]

**Review by stephanosio — APPROVED** [review]

**Pull request merged**